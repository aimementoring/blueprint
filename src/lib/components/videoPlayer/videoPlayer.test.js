import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './videoPlayer';

describe('Video', () => {
  it('renders properly', () => {
    const tree = renderer.create(<VideoPlayer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
