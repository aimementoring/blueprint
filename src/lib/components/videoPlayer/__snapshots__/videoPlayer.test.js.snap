// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video renders properly 1`] = `
<div
  className="theme-purple"
>
  <div
    className=""
  >
    <div
      className="react-plyr"
      data-plyr-provider="youtube"
    />
  </div>
</div>
`;
